sdk/python/lib/pulumi/automation/_stack.py

```diff
@@ -88,7 +88,8 @@ def __init__(self,
         self.version = version
         self.config: ConfigMap = {}
         for key in config:
-            self.config[key] = ConfigValue(**config[key])
+            config_value = config[key]
+            self.config[key] = ConfigValue(value=config_value["value"], secret=config_value["secret"])
```

Comment:
What was the error message before, did config[key] pass some unexpected parameters? Are "value" and "secret" fields required and always present? Especially "secret"? If it's optional consider `config_value.get("secret", None)`.

Reply:
Yeah, both `secret` and `value` are always present and required. Merged so we could get this in 3.1. 

The error was `TypeError: __init__() got an unexpected keyword argument 'objectValue'` because when the config is a map or an array it includes `objectValue` which is the deserialized object. I think we probably ideally want to include that as a way of resolving #5506 - but for now for consistency with other SDKs, I've removed it.

Reply:
That's the right change then, thank you!!